README.md

```diff
@@ -1 +1,47 @@
-# Awesome-Book
\ No newline at end of file
+# Awesome-Book
+
+> This is a basic website that allows users to add/remove books from a list. we archived this by using JavaScript objects and arrays. Also dynamically modifing the DOM and adding basic events.
+
+## Built With
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo (if available)
```

Comment:
- [ ] [OPTIONAL] Please either add  a live demo link or remove this sectio completely to make your code neat

Reply:
got it coach thank you for your CR

Reply:
You're welcome again 👌👌